src/pybind/mgr/crash/module.py

```diff
@@ -80,7 +80,7 @@ def _refresh_health_checks(self):
                 '%s crashed on host %s at %s' % (
                     crash.get('entity_name', 'unidentified daemon'),
                     crash.get('utsname_hostname', '(unknown)'),
-                    crash.get('timestamp', 'unknown time'))
+                    self.user_friendly_timestamp(crash.get('timestamp', 'unknown time')))
```

Comment:
nit: A little cleaner to me looks to just pass `crash.get('timestamp')` to `user_friendly_timestamp` and make `user_friendly_timestamp` return "unknown time" if `not timestr`.